### DISCUSSION TITLE
Row selection and expand column width

### DISCUSSION THREAD

**Discussion by viktorelo** [KevinVandy/material-react-table#205] [Q&A] [answered]
Hey, how can I customize the width of the selection and expand columns beside the default column size props? Couldn't find any related props.

edit: best approach I tried was to set the selection and expand columns' width through the defaultColumn prop, and set a "default" size on all of the data columns.

**Comment by KevinVandy** [comment, marked as answer]
There is a way to customize specific display columns with the `displayColumnDefOptions` prop.

Docs here: https://www.material-react-table.com/docs/guides/display-columns

Pretty much want something like this

```ts
<MaterialReactTable
  //...
  displayColumnDefOptions={{
    'mrt-row-select': {
      size: 60,
      //...
    }
  }}
/>
```
